### DISCUSSION TITLE
hi - About the Pages

### DISCUSSION THREAD

**Discussion by almarshad** [mindstellar/shopclass#455] [Ideas]
hello
i'm Aziz and im new here , been using Osclass for long 

i love your version of osclass to be honest and there is nothing wrong about it

my suggestion is about the Pages after activating the Permalinks

is it possible to make the Pages specially the internal name of the Pages in 2 languages like what you did in the SLUGs for the categories ??

that would be really helpful in arabic language..

thank you

**Comment by navjottomer** [comment]
Opened a feature request on this.
https://github.com/mindstellar/Osclass/issues/456
